app/uk/gov/hmrc/eusubsidycompliance/models/json/digital/package.scala

```diff
@@ -18,50 +18,37 @@ package uk.gov.hmrc.eusubsidycompliance.models.json
 
 import cats.implicits._
 import play.api.libs.json._
-import uk.gov.hmrc.eusubsidycompliance.models.json.eis.{Params, RequestCommon}
-import uk.gov.hmrc.eusubsidycompliance.models.types.EisAmendmentType.EisAmendmentType
+import uk.gov.hmrc.eusubsidycompliance.models.json.eis.Params
 import uk.gov.hmrc.eusubsidycompliance.models.types.Sector.Sector
-import uk.gov.hmrc.eusubsidycompliance.models.types.{EORI, EisAmendmentType, IndustrySectorLimit, UndertakingName, UndertakingRef}
-import uk.gov.hmrc.eusubsidycompliance.models.{BusinessEntity, Undertaking, UndertakingBusinessEntityUpdate}
+import uk.gov.hmrc.eusubsidycompliance.models.types.{IndustrySectorLimit, UndertakingName, UndertakingRef}
+import uk.gov.hmrc.eusubsidycompliance.models.{BusinessEntity, Undertaking}
 
 import java.time.format.DateTimeFormatter
-import java.time.{LocalDate, ZonedDateTime}
+import java.time.{Clock, Instant, LocalDate, LocalDateTime, ZoneOffset, ZonedDateTime}
 
 package object digital {
 
-  private val dateFormatter: DateTimeFormatter = DateTimeFormatter.ISO_LOCAL_DATE
+  val clock: Clock = Clock.systemUTC()
+  val formatter: DateTimeFormatter = DateTimeFormatter.ISO_INSTANT
+  val OK = "OK"
```

Comment:
Would an enum be appropriate here?

Reply:
I copy pasted some of the stuff as it is, but again I can look at it

Reply:
Maybe something to do in another ticket if it looks like a lot of work? Doesn't have to be changed now.

Reply:
sure. I will circle back to this after analysis on stub(if needed) for formatting changes. Thanks